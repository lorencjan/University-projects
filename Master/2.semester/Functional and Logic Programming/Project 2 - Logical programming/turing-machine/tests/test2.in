S a A a
A a B R
B b C L
C a F a
ab