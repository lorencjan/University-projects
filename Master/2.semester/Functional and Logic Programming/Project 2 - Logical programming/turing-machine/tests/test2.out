Sab
Aab
aBb
Cab
Fab
